### DISCUSSION TITLE
Is there a way to customize the script entry element?

### DISCUSSION THREAD

**Discussion by tobiasquadflieg** [nuxt/nuxt#22664] [Questions]
We are using Cookiebot to do the GDPR Stuff. The problem, cookiebot blocks the load of the entry.js
 `<script type="module" src="/_nuxt/entry.dbc187f5.js" crossorigin="" data-cookieconsent="preferences"></script>` 
because we use the nuxtjs/i18n that sets an cookie to save the users lang preferences. 

the easiest way to solve this is, just set the data-cookieconsent="ignore". But how do i customize the script tag? is it possible?

**Comment by TheAlexLichter** [comment]
The easiest way would be using a Nitro hook (in e.g. a Nitro plugin) and alter the html.
General example found in https://github.com/nuxt/nuxt/issues/18376#issuecomment-1438303970
